src/infobar.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react';
+import {
+  AppRegistry,
+  StyleSheet,
+  Text,
+  Image,
+  View
+} from 'react-native';
+
+export default class InfoBar extends Component {
+  constructor(props) {
+    super(props);
+  }
+  renderPicture(){
+    return(
+      <View style={styles.picture}>
+        <Image source={require('./img/bogdan.jpg')} style={{height: 30, width: 30, borderRadius: 30/2,}}/>
```

Comment:
1) The style should live under StyleSheet.create({
2) We need to make the infobar dynamic, to change photo and name based on the props received from the parent. For example, instead of hardcoding ```./img/bogdan.jpg```, let's use a prop called ```this.props.avatarUrl```, and load the avatar like this: ```source={{uri: this.props.avatarUrl}}```